lib/luatest_server.py

```diff
@@ -0,0 +1,110 @@
+import glob
+import os
+import re
+import sys
+from subprocess import STDOUT
+from gevent.subprocess import Popen, PIPE
+
+from lib.preprocessor import TestState
+from lib.sampler import sampler
+from lib.server import Server
+from lib.tarantool_server import Test
+from lib.tarantool_server import TarantoolServer
+
+
+class LuatestTest(Test):
+    """ Handle *_test.lua.
+
+    Provide method for executing luatest <name>_test.lua test.
+    """
+
+    def __init__(self, *args, **kwargs):
+        Test.__init__(self, *args, **kwargs)
+        self.valgrind = kwargs.get('valgrind', False)
+
+    def execute(self, server):
+        """Execute test by luatest command
+
+        Execute 'luatest -v <name>_test.lua -o tap --shuffle none'
+        Provide a verbose output in the tap format.
+        Use shuffle option in none mode for avoiding mixing tests.
+        Use capture mode.
+        """
+        server.current_test = self
+        ts = TestState(self.suite_ini, None, TarantoolServer,
+                       self.run_params,
+                       default_server_no_connect=server)
+        self.inspector.set_parser(ts)
```

Comment:
I'm not sure that we want to use `require('test_run').new():cmd(<...>)` functions in tests written for luatest:

1. It'll add hard dependecy on test-run for tests and block transition to pure luatest (I don't know, whether we'll actually want it, but I want to have this ability).
2. If it is necessary and works, I want to see a test in the test-run repository.

If you found that it is necessary in our tests, please, point me an example.

Reply:
As I remember, for now, I erased all test_run dependencies